dist/index.html

```diff
@@ -7,29 +7,35 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;1,600&display=swap"
     rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/games" rel="stylesheet">
   <title>Leaderboard</title>
 <script defer src="runtime.js"></script><script defer src="main.js"></script></head>
 
 <body>
-  <h1>LeaderBoard</h1>
+  <h1 class="main-head">LeaderBoard</h1>
+  <div class="bg">
+    <h1>LeaderBoard</h1>
+  </div>
+  <div class="bg bg2"></div>
+  <div class="bg bg3"></div>
+<div class="content">
   <main class="leaderboard">
     <div class="container">
       <div class="recent">
         <h2>Recent score</h2>
         <button id="refresh">Refresh</button>
       </div>
       <ul class="score-list">
-      
       </ul>
     </div>
-
     <div class="form">
       <h2>Add your score</h2>
       <input type="text" id="name" name="name" placeholder="Enter your name" required>
       <input type="number" id="score" name="score" placeholder="Enter your score" required>
       <button id="submit">Submit</button>
     </div>
   </main>
```

Comment:
- [ ] You did have a beautiful design 👏 . However, please adjust the submit button and the position of the form to look exactly like the layout given on the wireframe, to complement the good work you've done so far 👍 
- [ ] Though you might be of the opinion that the current design is "great", the suggestion to stick to requirements is a simulation of what your hiring manager and potential clients will ask for. o please check the image comparison below and adjust it appropriately 

## Wireframe:
![image](https://user-images.githubusercontent.com/50990925/224045732-81331854-963f-411a-a3cf-ccf1d0b1ae3b.png)

## Your view:
![image](https://user-images.githubusercontent.com/50990925/224046158-d93f9382-77f9-47a5-8855-edf8c143397d.png)

Reply:
Hi 👋 @Shedrack-Sunday , 
Hope all is well with you ☺️,
Thanks for your efforts in reviewing my code,
I'll adjust the submit button and the position of the form, as you asked  :+1: .
Best Regards

Reply:
You're welcome @MostafaShahat2021🤗